src/main/scala/djinni/Main.scala

```diff
@@ -97,7 +98,9 @@ object Main {
       }
 
       override def showUsageOnError = false
+      head("djinni generator version", Main.getClass.getPackage.getImplementationVersion)
       help("help")
+      version("version")
```

Comment:
the scopt library already automatically supports the `--version` parameter. All you need is to define a `head()` with the version information.